components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher.v1.common/src/main/java/org/wso2/carbon/apimgt/rest/api/publisher/v1/common/mappings/PublisherCommonUtils.java

```diff
@@ -333,16 +333,18 @@ public static API updateApi(API originalAPI, APIDTO apiDtoToUpdate, APIProvider
 
         //attach micro-geteway labels
         assignLabelsToDTO(apiDtoToUpdate, apiToUpdate);
+        String tenantDomain =  CarbonContext.getThreadLocalCarbonContext().getTenantDomain();
```

Comment:
can we use RestAPIUtil.getLoggedInUserTenantDomain?

Reply:
https://github.com/wso2/carbon-apimgt/pull/9658/commits/142cd9cad67662fc64ac6e886127be649322fd43